p2-feature/org.eclipse.collections.feature/pom.xml

```diff
@@ -15,7 +15,7 @@
 	<parent>
 		<groupId>org.eclipse.collections</groupId>
 		<artifactId>p2-feature</artifactId>
-		<version>9.1.0-SNAPSHOT</version>
+		<version>9.2.0</version>
```

Comment:
This has to be synchronized with the version number in feature.xml and the category.xml file version in the repository project.